wikitemplate.md

```diff
@@ -145,6 +165,10 @@ _Each start should take less than 7 seconds_
   - [ ] Verify tips list is not lost after upgrade
   - [ ] Verify wallet panel transactions list is not lost after upgrade
 
+### Social-media blocking settings
+
+- [ ] Verify individual `Social media blocking` buttons works as intended when enabled/disabled
```

Comment:
Could we also list the original page we used for testing these - so we don't have to go digging for a place to test?

Reply:
ok will add the link for the tests